plugins/com.google.cloud.tools.eclipse.appengine.localserver/src/com/google/cloud/tools/eclipse/appengine/localserver/ui/CloudSdkRuntimeWizardFragment.java

```diff
@@ -146,18 +145,23 @@ public void widgetSelected(SelectionEvent event) {
       }
     });
 
-    File location = new CloudSdkProvider(null).getCloudSdkLocation();
-    if (location != null) {
-      dirTextBox.setText(location.toString());
+    CloudSdk sdk = new CloudSdkProvider().getCloudSdk();
+    if (sdk != null) {
+      java.nio.file.Path location = sdk.getJavaAppEngineSdkPath();
+      if (location != null) {
```

Comment:
@elharo location is never null because the library never returns null. But should we have the null check regardless?